pandas/__init__.py

```diff
@@ -10,11 +10,10 @@
     try:
         __import__(dependency)
     except ImportError as e:
-        missing_dependencies.append(dependency)
+        missing_dependencies.append("{0}: {1}".format(dependency, str(e)))
 
 if missing_dependencies:
-    raise ImportError(
-        "Missing required dependencies {0}".format(missing_dependencies))
+    raise ImportError("Unable to import required dependencies:\n" + "\n".join(missing_dependencies))
```

Comment:
can you do this like
```
raise ImportError("Unable to import required dependencies:\n" 
                              "\n".join(missing_dependencies))
```

since inside the parens you don't need the +

Reply:
That gives a different output - the current one joins the dependencies with `\n`:
```Unable to import required dependencies:
a
b
c
```

Whereas without the "+" it joins them with the entire error message and two newlines:
```aUnable to import required dependencies:

bUnable to import required dependencies:

c
```